ccd-definition/FixedLists/CareSupervision/ManageOrdersType.json

```diff
@@ -62,18 +62,25 @@
     "ListElement": "Interim supervision order (C35B)",
     "DisplayOrder": 9
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ManageOrdersType",
+    "ListElementCode": "C45A_PARENTAL_RESPONSIBILITY_ORDER",
```

Comment:
If you are moving it here, you want to delete it from ManageOrdersUploadType

Reply:
moved